torch/_torch_docs.py

```diff
@@ -7296,11 +7296,9 @@ def merge_dicts(*dicts):
 The :attr:`std` is a tensor with the standard deviation of
 each output element's normal distribution
 
-The shapes of :attr:`mean` and :attr:`std` don't need to match, but the
-total number of elements in each tensor need to be the same.
-
-.. note:: When the shapes do not match, the shape of :attr:`mean`
-          is used as the shape for the returned output tensor
```

Comment:
Nice to have this deprecated, even if it was 4 versions later than we said we would deprecate it (cc @mruberry)